blockchain/v0/reactor_test.go

```diff
@@ -118,8 +125,13 @@ func newBlockchainReactor(
 
 		thisBlock := makeBlock(blockHeight, state, lastCommit)
 
+		err = thisBlock.PutBlock(context.Background(), ipfsAPI.Dag())
+		if err != nil {
+			panic(err)
+		}
+
 		thisParts := thisBlock.MakePartSet(types.BlockPartSizeBytes)
-		blockID := types.BlockID{Hash: thisBlock.Hash(), PartSetHeader: thisParts.Header()}
+		blockID := types.BlockID{Hash: thisBlock.Hash(), PartSetHeader: thisParts.Header(), DataAvailabilityHeader: &thisBlock.DataAvailabilityHeader}
```

Comment:
Just asking: will this PR remove the `PartSetHeader` from the `BlockID` or will this rather happen in a separate PR?

Reply:
that will likely be in #287